### ISSUE TITLE
Geosearch

### ISSUE THREAD

**Issue by olafveerman** [developmentseed/worldpop#27]
A geosearch field could come in handy to quickly pan and zoom to a location. Probably only interesting when we have more data.

**Comment by anandthakker** [comment]
Yeah, this would be very nice.  We should have much more data in there by the end of the week, so I think this would be worth adding soon.

**Comment by anandthakker** [comment]
One thought: our data pipeline is generating straightforward coverage data that's probably small enough to bundle with the javascript.  I wonder if we could use that to filter the geosearch to places where data's available.
